vhdbuilder/packer/install-dependencies.sh

```diff
@@ -52,12 +52,31 @@ if [[ ${UBUNTU_RELEASE} == "18.04" ]]; then
   disableSystemdTimesyncdAndEnableNTP || exit 1
 fi
 
+if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
+  echo "VHD will be built with containerd as the container runtime"
+  CONTAINERD_VERSION="1.4.1"
+  installStandaloneContainerd
```

Comment:
I could be wrong but I thought this function is not always available?

Reply:
Yea that's what I thought too but packer is actually able to import go-templated bash functions as well. I think packer was already using a few of these functions before this PR.